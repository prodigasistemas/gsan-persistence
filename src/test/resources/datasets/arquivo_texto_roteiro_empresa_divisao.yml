micromedicao.arq_txt_roteiro_emp_div:
  - tred_id: 1
    txre_id: 1
  - tred_id: 2
    txre_id: 2
  - tred_id: 3
    txre_id: 1
  - tred_id: 4
    txre_id: 3