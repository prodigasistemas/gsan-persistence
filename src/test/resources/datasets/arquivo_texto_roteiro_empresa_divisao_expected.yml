micromedicao.arq_txt_roteiro_emp_div:
  - tred_id: 2
    txre_id: 2
  - tred_id: 4
    txre_id: 3